config.js

```diff
@@ -37,7 +38,7 @@ config.freegeoip.port = 8080;
  * Redis server
  */
 config.redis.host = '127.0.0.1';
-config.redis.port = 6379;
+config.redis.port = 6380;
```

Comment:
Should this be changed?

Reply:
No, it should be 6379 there. I'll make a fix for this.